backend/game/cards/keep_cards/health_manipulation_cards/test_it_has_a_child.py

```diff
@@ -0,0 +1,28 @@
+from game.cards.keep_cards.health_manipulation_cards.it_has_a_child import ItHasAChild
+from game.values import constants
+
+
+def test_it_has_a_child_costs_7_energy():
+    assert ItHasAChild().cost == 7
+
+
+def test_it_has_a_child_discards_all_cards_when_health_is_0(player):
+    player.add_card(ItHasAChild())
+    player.current_health = constants.DEATH_HIT_POINT
+    ItHasAChild().immediate_effect(player, None)
+    ItHasAChild().special_effect(player, None)
+    assert not player.has_instance_of_card(ItHasAChild())
+
+
+def test_it_has_a_child_loses_all_stars_when_health_is_0(player):
+    player.energy = constants.DEFAULT_ENERGY_CUBE
+    ItHasAChild().immediate_effect(player, None)
```

Comment:
this test doesn't work. Player starts with 0 victory points, runs the special effect of a card he doesn't have, then asserts he still has 0 victory points. Same with test below